### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by iranzo**
Fixes title with `'s` and KubeVirt spelling plus some extra spacing in code

### CODE REVIEW

File: _posts/2019-11-11-Access-Virtual-Machines-graphic-console-using-noVNC.markdown [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: post
 author: Yafei Bao
-description: Demonstrate how to access virtual machines' graphic console using noVNC.
+description: Demonstrate how to access virtual machine's graphic console using noVNC.
```
Comment (codificat) [inline _posts/2019-11-11-Access-Virtual-Machines-graphic-console-using-noVNC.markdown]: I believe the original was correct (plural, "virtual machines").

If you turn it to singular you probably need an additional article, like "a virtual machine".

Comment (iranzo) [reply _posts/2019-11-11-Access-Virtual-Machines-graphic-console-using-noVNC.markdown]: > I believe the original was correct (plural, "virtual machines").
> 
> If you turn it to singular you probably need an additional article, like "a virtual machine".

Let's revert this then, ok?

**Comment by codificat** [comment]
Yes - the two instances where the plural (virtual machines') was changed to
singular (virtual machine's). The apostrophe was correctly placed (after
the s) in the original plural version.

>

File: _posts/2019-11-11-Access-Virtual-Machines-graphic-console-using-noVNC.markdown (hunk 1 above)
Comment (codificat) [reply _posts/2019-11-11-Access-Virtual-Machines-graphic-console-using-noVNC.markdown]: Yes - the two instances where the plural (virtual machines') was changed to singular (virtual machine's). The apostrophe was correctly placed (after the s) in the original plural version.

Comment (codificat) [reply _posts/2019-11-11-Access-Virtual-Machines-graphic-console-using-noVNC.markdown]: Ah, I see it's already reverted in #412. thanks.

Comment (codificat) [reply _posts/2019-11-11-Access-Virtual-Machines-graphic-console-using-noVNC.markdown]: oh, although the revert rolled back all the changes - not only this particular plural/singular change...

**Pull request merged**